Doc/faq/programming.rst

```diff
@@ -1020,8 +1020,8 @@ For more complicated input parsing, regular expressions are more powerful
 than C's ``sscanf`` and better suited for the task.
 
 
-What does 'UnicodeDecodeError' or 'UnicodeEncodeError' error  mean?
--------------------------------------------------------------------
+What does ':exc:`UnicodeDecodeError`' or ':exc:`UnicodeEncodeError`' error  mean?
```

Comment:
Added exc roles to the exception names to help clarify for the translator this should not be translation, but this makes a hyperlink to the class. Add ! to not link or simply remove?